inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningRecommender.java

```diff
@@ -92,11 +93,23 @@ public RecommendationDifference generateRecommendationWithLowestDifference(Docum
         }
 
         // remove rejected recommendations
-        removeRejectedOrSkippedAnnotations();
+        removeRejectedOrSkippedAnnotations(true, learnSkippedRecommendationTime);
 
         return calculateDifferencesAndReturnLowestDifference ();
     }
 
+    public boolean hasRecommendationWhichIsSkipped()
+    {
+        getRecommendationFromRecommendationModel();
+        removeRejectedOrSkippedAnnotations(false, null);
+        if (listOfRecommendationsForEachToken.size() > 0) {
+            return true;
+        }
+        else {
+            return false;
+        }
+    }
```

Comment:
This could be done as 

```
return !listOfRecommendationsForEachToken.isEmpty();
```

The same applies to other places where something like this happens:

```
if (condition) {
  return true;
}
else {
  return false;
}
```

Reply:
have changed to directly return boolean condition to avoid if... else...